test/Microsoft.ML.Sweeper.Tests/SweeperTest.cs

```diff
@@ -0,0 +1,65 @@
+using Microsoft.ML.Runtime;
```

Comment:
You will see our other files tend to have a standard header:

```csharp
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

It may be a nice idea to include that as well in this file.